Rules/CommonScripts/ChatCommands.as

```diff
@@ -302,11 +303,16 @@ bool onServerProcessChat(CRules@ this, const string& in text_in, string& out tex
 					}
 					server_MakePredefinedScroll(pos, s);
 				}
-				else if(tokens[0] == "!coins")
+				else if (tokens[0] == "!coins")
 				{
 					int money = parseInt(tokens[1]);
 					player.server_setCoins(money);
 				}
+				else if (tokens[0] == "!sign")
+				{
+					CMap@ map = getMap();
+					createSign(pos, text_in.substr(tokens[0].length+1, -1));
```

Comment:
The default value for the second argument is -1 so this line can be simplified to the following:
```suggestion
					createSign(pos, text_in.substr(tokens[0].length + 1));
```